validator/main_validator.py

```diff
@@ -657,7 +657,10 @@ def str2metric(self, val, row_id, spread_sheet_name, wb_spreadsheet, field):
                 except:
                      if " " in val:
                         val, unit = val.split(" ", 1)
-                        current_metric['estimate'] = float(val)
+                        try:
+                            current_metric['estimate'] = float(val)
+                        except ValueError:
+                            self.report_error(spread_sheet_name, row_id, f'Failed to extract metric(s). Is the correct separator (;) used?')
```

Comment:
Should we include `val` into the report_error message ?

Reply:
val would include the float part of the first metrics + the rest of all the other metrics unsplitted. I guess that could give an additional clue about the problem.